relayer-engine/src/listener/eventHarness.ts

```diff
@@ -12,6 +13,24 @@ const logger = () => {
   return _logger;
 };
 
+const receivedEventsCounter = new Counter({
```

Comment:
nit: same as for executor, how about moving these metric definitions to `listener/metrics.ts`?